### PR TITLE
Create starter django project #2

### PR DESCRIPTION
**Pull request by YuliaShap**

### CODE REVIEW

**Review by Thoughtseize1 — APPROVED** [review]
The Django project is initialized.
I tested this branch, and everything works 🙌🏻
image

File: forum/settings.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-t&h(6bczgs&7s8to0qp5i)csn_izs%=t!*q(km=fhk4l@$p2xl'
```
Comment (VovaD199) [inline forum/settings.py:28]: Please take the secret key
SECRET_KEY = os.environ.get('SECRET_KEY')

File: forum/settings.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
```
Comment (VovaD199) [inline forum/settings.py:33]: ALLOWED_HOSTS = ["*"]

File: forum/settings.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.postgresql_psycopg2',
+#         'NAME': 'postgres1',
+#         'USER': 'postgres',
+#         'PASSWORD': 'postgres',
+#         'HOST': 'localhost',
+#         'PORT': '5432',
+#     }
+# }
```
Comment (VovaD199) [inline forum/settings.py:103]: Commented lines should be removed

**Review by VovaD199 — APPROVED** [review]

File: forum/settings.py (hunk 1 above)
Comment (YuliaShap) [reply forum/settings.py]: Fixed

File: forum/settings.py (hunk 2 above)
Comment (YuliaShap) [reply forum/settings.py]: Fixed

File: forum/settings.py (hunk 3 above)
Comment (YuliaShap) [reply forum/settings.py]: Fixed

**Review by superolegatron — APPROVED** [review]

**Pull request merged**